src/js/index.ts

```diff
@@ -32,9 +32,9 @@ export {
 
 // We need to import it so we patch the hub with global functions
 // aka. this has side effects
-import '@sentry/tracing';
+import '@sentry/browser';
```

Comment:
This might need `@sentry-internal/tracing` I need to check the JS implementation.